src/layouts/auth/sign-in-5/index.tsx

```diff
@@ -41,6 +42,18 @@ export default ({ navigation }): React.ReactElement => {
     setSMSCodeVisible(!smsCodeVisible);
   };
 
+  const renderIconPassword = (props): ReactElement => (
+    <TouchableWithoutFeedback onPress={onPasswordIconPress}>
+      <Icon {...props} name={'lock'} />
```

Comment:
Let's omit braces where possible
```suggestion
      <Icon {...props} name='lock' />
```